### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by PisarskyiDev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,12 @@ So in this task requirements are next:
   And to pass environment variables to docker container use `-e` flag.
 - Don't forget to add `.dockerignore` file to your PR;
 - You must *modify* this line with correct command to pull your image:
-  COMMAND=`docker pull <YOUR_DOCKER_ID/YOUR_IMAGE_NAME>`.
+
+  COMMAND=`docker pull megah1t/weather-api>`.
```
Comment (Bohdan-Salamakha) [inline README.md]: I can't pull your repo
fix an issue
image

File: app/main.py [hunk 2]
```diff
@@ -1,6 +1,29 @@
+import os
+import requests
+
+from dotenv import load_dotenv
+
+load_dotenv()
```
Comment (Bohdan-Salamakha) [inline app/main.py]: place it inside of if __name__ == "__main__":

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]
I can get the weather info without an API_KEY
The logic is incorrect
image
+
Paris isn't the same as Odessa, change the city
https://github.com/mate-academy/py-docker-weather-api#:~:text=getting%20current%20weather-,for%20Paris%20city,-.%20But%20the%20most

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]
I still can use your app without an API_KEY!
fix it
image

**Review by YukihiroSM — APPROVED** [review]

**Pull request closed without merging**